README

```diff
@@ -269,14 +269,17 @@ covered.
 # The Cassini and Huygens models are by Jack Higgins:
     http://homepage.eircom.net/~jackcelestia/
 
-# The 3D model of ISS is a deconstruction of Andrew Farnaby's
-  complete ISS model with textures by Bob Hundley. The model
-  represents the ISS as of June 2008 and is a modification of
-  the model that can be found here:
-    http://www.celestiamotherlode.net/catalog/show_addon_details.php?addon_id=1199
-   ISS model of June 2008 (with Kibo) by krisci3 (modified and
-   converted in order to work with JPG by Ulrich Dickmann, a.k.a.
-   Adirondack)
+# The 3D model of ISS is a representation of the station created by NASA-VTAD
+  on April 22, 2019:
+    https://science.nasa.gov/resource/international-space-station-3d-model/
+  The configuration shown is what the ISS looked like from May
+  2011 until February 2015, shortly after STS-134 (Endeavour)
+  delivered the AMS and ELC-3 pallets to the station. Model and textures were
+  converted for use in Celestia by DaveBowman2001
+  Orbital elements used were retrieved from JPL HORIZONS using
+  Epoch 2024-Nov-20 (JD 2460634.5), i.e. the station's 26th anniversary since
+  launch:
+    https://ssd.jpl.nasa.gov/horizons/app.html#/
```

Comment:
I'd remove this part, since data is not usually attributed in the README

Reply:
Done da60167e3490976976bad9bebaa60b21a708efb3